src/Model/ChangeStreamIterator.php

```diff
@@ -199,6 +197,7 @@ public function next(): void
 
         try {
             parent::next();
+
```

Comment:
From SlevomatCodingStandard.Classes.ParentCallSpacing.

I don't feel strongly about this and it was a small change affecting only this file.